libr/arch/p/avr/pseudo.c

```diff
@@ -209,8 +209,12 @@ static int parse(RParse *p, const char *data, char *str) {
 }
 
 RParsePlugin r_parse_plugin_avr_pseudo = {
-	.name = "avr.pseudo",
-	.desc = "AVR pseudo syntax",
+	.meta = {
+		.name = "avr.pseudo",
+		.desc = "AVR pseudo syntax",
+		.author = "pancake",
+		.license = "LGPL3",
```

Comment:
Please use SPDX namings. See the related commits in git log | grep SPDX for more info